resources/definitions/fdmprinter.def.json

```diff
@@ -5144,7 +5144,7 @@
                         "support_top_distance":
                         {
                             "label": "Support Top Distance",
-                            "description": "Distance from the top of the support to the print.",
+                            "description": "Distance from the top of the support to the print. Note that this is not rounded.",
```

Comment:
In this case I would not mention it because this is what one would expect. But too much description does not hurt..

Reply:
Feel free to remove it :-)